README.md

```diff
@@ -149,6 +149,18 @@ A command line flag exists `--podCheckNamespaces` which can optionally contain a
 - Error state toleration: 5 minutes
 - Check name: `podStatus`
 
+#### DNS
+
+Checks for failures with DNS, including resolving within the cluster and outside of the cluster. Default endpoints to resolve: kubernetes.default, aws.amazon.com, cloud.dns.com
+
+A command-line flag exists `--dnsEndpoints` which can optionally include a comma separated list of DNS endpoints to test. 
+
+- Timeout: 1 minutes
```

Comment:
If we are going to have such a short error toleration I'd recommend a faster interval.  4-5 checks per minute should be about right.  This is a really light weight check so I don't thing its a problem to check often.